zp-relayer/pool.ts

```diff
@@ -72,6 +76,9 @@ class Pool {
   constructor() {
     this.PoolInstance = new web3.eth.Contract(PoolAbi as AbiItem[], config.poolAddress)
 
+    this.treeParamsHash = Pool.getHash(config.treeUpdateParamsPath)
+    this.transferParamsHash = Pool.getHash(config.transferParamsPath)
```

Comment:
Transfers params file is quite large (~70MB). How much time it takes to calculate it hash? Does it make sense to calculate the hash only once and keep it cached?

Reply:
These hashes are calculated once and stored during initialisation of the Pool instance.
Total time to calculate these should not exceed 0.2-0.4 sec